scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py

```diff
@@ -296,6 +299,8 @@ def test_verbosity(tmpdir):
         _, _ = lobpcg(A, Q, maxiter=3, verbosityLevel=9)
 
 
+@pytest.mark.xfail(_IS_32BIT and sys.platform == 'win32',
+                   reason="tolerance violation on windows")
```

Comment:
There was a comment in the cross-listed issue that 32-bit build are "niche" so it is not worth debugging. IIRC, 32-bit builds were considered quite common on Windows up until quite recently,  with 3.10 support for 32-bit falling away on a lot of services I guess.

Still, there's already another `xfail` marker for this test, which makes me a bit less nervous I suppose ("the test has issues sometimes" seems fair).

Reply:
> There was a comment in the cross-listed issue that 32-bit build are "niche" so it is not worth debugging.

That's likely just my personal biases showing. conda-forge has not supported 32bit builds for several years, and almost all projects I pay attention to have done the same. That being said, I vaguely remember that until recently python installations on windows defaulted to 32bit. I stand by the "not worth debugging though", especially as it's only a tolerance violation, and there are other skips for this test already.